spec/features/talks/search_spec.rb

```diff
@@ -0,0 +1,36 @@
+RSpec.describe 'Talks search' do
+  let(:event) { create :event, title: 'Event Title' }
+
+  let!(:talk_1) { create :talk, title: 'Talk 1', description: "infrastructure", event: event }
```

Comment:
`let!(:talk_1) { create :talk, title: title_of_talk_1', description: "infrastructure", event: event }`
and cetera